src/apps/users/tests.py

```diff
@@ -0,0 +1,25 @@
+from apps.users.domain import UserCreate
+from apps.users.services import UsersCRUD
+from infrastructure.test import BaseTest, rollback
+
+
+class TestApplet(BaseTest):
+    fixtures = ["users/fixtures/users.json"]
+
+    @rollback
+    async def test_creating_user(self):
+        user = UserCreate(
+            email="qwe@mail.ru",
+            full_name="tom isaak",
+            hashed_password="qweqweqwe",
+        )
+        user, _ = await UsersCRUD().save_user(user)
+        existed_user = await UsersCRUD().get_by_id(1)
+        assert user.id == 2
+        assert existed_user.id == 1
+        assert existed_user.full_name == "Tom Isaak"
+
+    @rollback
+    async def test_checking_fixture_count(self):
+        count = await UsersCRUD().count()
+        assert count == 1
```

Comment:
1. I think the naming of this class is wrong, don't you think?
2. Why don't you use pytest fixtures power?

I would suggest avoid using classes if it is not needed and simplify it to the next code

```python
# apps.users.tests.factories.py
from pydantic_factories import ModelFactory
class UserCreateFactory(ModelFactory):
    __model__ = UserCreate


# apps.users.tests.conftest.py
@pytest.fixture
def user_create_schema() -> UserCreate:
    return UserCreateFactory.build()


# apps/users/tests/test_services.py
@rollback
async def test_user_create(user_create_schema):
    user, created = await UsersCRUD().save_user(user_create_schema)
    assert user.id == 2
    assert created is True

```

Reply:
I try to avoid using `@pytest.fixture` for getting fixtures as an argument because when tests need much data, it is hard to generate(manage test inputs), if you use fixtures(database as json/yaml) it is faster to insert into database and separated from codebase

Reply:
Data generates automatically. Check the example with the factory class

Reply:
I know that it generates automatically. The main weird thing is that you have to deal dynamic data which you don't manage and it is written as a code. When it is simple static data, it is more convenient to see static behavior of method. I have been introduced to project where there were many fixtures(@pytest.fixture) and they were related to each other, I have faced with issue which was to understand all fixtures and how they related(and they were dynamic which means I could not see their information). I advised them to group them.

Reply:
and `save_user` method returns `True` in any case or exception
<img width="656" alt="image" src="https://user-images.githubusercontent.com/17402086/206469717-0c84d0ec-f176-4123-8b92-f9c2d9c9e4bc.png">